src/display.cpp

```diff
@@ -3,7 +3,7 @@
 // - Aellen Quentin
 
 #include <iostream>
-#include <stdlib.h>
```

Comment:
Hy, I think it is used for system function calls (UNIX). 
You can include ```#include <cstdlib>``` to be sure.

Reply:
Added.